client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.js

```diff
@@ -159,6 +161,30 @@ const currentLineWithoutOutputVariableName = (
   return currentLine;
 };
 
+const returnMatchingCombination = (rpaTask, allMatchingCombinations) => {
```

Comment:
documentation is missing

Reply:
Thanks for the hint. Now there is a documentation